src/Mapper.js

```diff
@@ -2375,11 +2391,8 @@ export default Component.extend({
     const _opts = utils.pick(opts, ['existingOnly'])
     if (utils.isArray(record)) {
       const errors = record.map((_record) => schema.validate(_record, utils.pick(_opts, ['existingOnly'])))
-      const foundErrors = errors.filter((err) => err)
-      if (foundErrors.length) {
-        return errors
-      }
-      return undefined
+
+      return errors.some(Boolean) ? errors : undefined
```

Comment:
By the way, I've just found out that `Array.isArray` is also a part of ES5 spec :) http://www.ecma-international.org/ecma-262/5.1/#sec-15.4.3.2